### DISCUSSION TITLE
How do you delete a list you have created

### DISCUSSION THREAD

**Discussion by ghost** [community/community#37467] [Other Feature Feedback, Questions, & Ideas] [answered]
How do you delete a list? I've looked everywhere but I can't find a delete button

**Comment by radar-blip** [comment, marked as answer]
Open the list you want to delete and click on the `edit list` button, from there you can delete.

**Reply by DSDmark** [reply]
which list he is talking about?

**Reply by radar-blip** [reply]
Lists that appear when you click on `Stars`  

![lists](https://user-images.githubusercontent.com/39196767/198230157-6510e929-20f2-4b4d-ad0f-aaa05d570a73.png)

**Reply by DSDmark** [reply]
O I got it.

**Reply by Tuanhuu773** [reply]
Số thẻ

**Reply by wanniethecharacter** [reply]
thankkkkkk youuuuuuuuuuuuuuuuuuuuuuuuuuuuu!!!!!!!!!!!!!

**Comment by Tuanhuu773** [comment]
82292a68579e7255885e7b6fb310378f91bbf4c0
